cylc/rose/stem.py

```diff
@@ -23,25 +23,53 @@
 
 To run a rose-stem suite use "cylc play".
 
-Looks for a suite to install either in $PWD/rose-stem, or in a specified
-location - e.g:
-
-# Install a rose-stem suite from PWD/rose-stem
-$ rose stem
-
-# Install a rose-stem suite from PWD/relative-path.
-$ rose stem relative-path
-
-# Install a rose-stem suite from specified abs path.
-$ rose stem /absolute/path
+Install a suitable suite with a specified set of source tree(s).
+
+Default values of some of these settings are suite-dependent, specified
+in the `rose-suite.conf` file.
+
+Examples
+
+    rose stem --group=developer
```

Comment:
Do you think out users will understand the --group option? It used to appear in the help and is now not explained?

Reply:
You've spotted an aspect of copying the Rose 2019 help that I forgot. 😞

Reply:
I've re-added the help for those things - I've also put the rose-stem specific options into a group so that they get their own sub heading.